tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -138,12 +141,42 @@ public static void initTrace(Optional<Boolean> isObservable, String traceId) {
                 .map(observable -> observable ? Observability.SAMPLE : Observability.DO_NOT_SAMPLE)
                 .orElse(Observability.UNDECIDED);
 
-        setTrace(createTrace(observability, traceId));
+        setTrace(createTrace(observability, traceId, Optional.empty()));
     }
 
-    /** Initializes the current thread's trace, erasing any previously accrued open spans. */
+    /**
+     * Initializes the current thread's trace, erasing any previously accrued open spans.
+     *
+     * @deprecated Use {@link #initTraceWithSpan(Observability, String, String, SpanType)}
```

Comment:
there seems to be relatively little usage of this method, with the majority of it being in tests

Reply:
Unfortunately many tests attempt to peel back layers of abstraction rather than testing results. These methods are only really used for server-incoming requests where we have prior knowledge of a request identifier.

Reply:
Sorry my comment was unclear, I was looking at internal usage across the fleet

Reply:
Ah, that makes sense :+1: